music/src/piano_genie/model.ts

```diff
@@ -186,14 +223,15 @@ class PianoGenie {
    * application in real time (it keeps track of time internally).
    *
    * @param button Button number (one of {0, 1, 2, 3, 4, 5, 6, 7}).
+   * @param chord Chord type.
    * @param temperature Temperature. From 0 to 1, goes from argmax to random.
    * @param seed Random seed. Use a fixed number to get reproducible output.
    */
-  next(button: number, temperature?: number, seed?: number) {
+  next(button: number, chord: Chord, temperature?: number, seed?: number) {
```

Comment:
(as per below comment), i think this should be `chord=NOCHORD` as a default, and moved to the end